python/nano/test/test_trainer_ipex.py

```diff
@@ -55,13 +73,44 @@ def test_resnet18_ipex(self):
             use_orca_lite_trainer=True)
 
     def test_trainer_compile(self):
-        model = ResNet18(10, pretrained=False, include_top=False, freeze=True)
-        loss = nn.CrossEntropyLoss()
-        optimizer = torch.optim.Adam(model.parameters(), lr=0.01)
         trainer = Trainer(max_epochs=1)
-        pl_model = Trainer.compile(model, loss, optimizer)
-        train_loader = create_data_loader(data_dir, batch_size, num_workers, data_transform)
-        trainer.fit(pl_model, train_loader)
+        pl_model = Trainer.compile(self.model, self.loss, self.optimizer)
+        trainer.fit(pl_model, self.train_loader)
+
+    def test_trainer_quantize_inc_ptq_compiled(self):
+        # Test if a Lightning Module compiled by nano works
+        train_loader_iter = iter(self.train_loader)
+        trainer = Trainer(max_epochs=1)
+        pl_model = Trainer.compile(self.model, self.loss, self.optimizer)
+
+        # Case 1: Default
+        qmodel = trainer.quantize(pl_model, self.train_loader)
+        assert qmodel
+        out = qmodel(next(train_loader_iter)[0])
+        assert out.shape == torch.Size([256, 10])
```

Comment:
This there a way to check if the model is actually quantized?

Reply:
Save the model and see if model size is smaller. But actually if `qmodel` is not `None`, then the model should be quantized, otherwise it should return `None`.

Reply:
We may add a test at the end of `trainer.quantize` and raise error once it's failed? Since return a None can be a "silence failure", which is unfriendly.